### DISCUSSION TITLE
[mpvnet] 简易包 obs_fin

### DISCUSSION THREAD

**Discussion by hooke007** [hooke007/mpv_PlayKit#261] [libmpv]
<s>如无意外，最后一版。</s>
https://github.com/hooke007/mpv.net_CM/releases/tag/6032re

已决定中止后续的维护支持。

**Comment by MisakaSakura** [comment]
如果我想用lite版设置替代这版。我该怎么做来完全重置配置呢不想？轻微强迫症用户不喜欢重复的文件

**Reply by hooke007** [reply]
参考两个软件的手册自行探索。
也做不到完全适用，功能上存在局部不兼容。

**Comment by diana7127** [comment]
请问如何在mpv.net_CM使用thumbfast？
有看过这篇[https://github.com/mpvnet-player/mpv.net/issues/520](url)，已在.NET/process-instance改为multi多线程，并根据@tsl0922的设置将local mpv_path = "mpv" 改为 local mpv_path = "mpvnet"，进度条上方的缩略图框仍为白色，但会创建一个新进程显示缩略图，在进度条上移动光标位置，新进程（缩略图）也会随之改变。请问有什么办法可以修复吗？
或者有其他使用thumbfast的设置吗？
谢谢

**Reply by diana7127** [reply]
![Snipaste_2023-02-18_01-05-38](https://user-images.githubusercontent.com/125502871/219686789-8c3ff45b-8850-43fc-af67-fcaaf1c32480.jpg)

**Reply by hooke007** [reply]
mpvnet的内核就不支持内编。改代码没用。塞一个mpv.exe 进mpvnet目录就行

**Reply by hooke007** [reply]
相关 https://github.com/hooke007/MPV_lazy/discussions/18#discussioncomment-3700782

**Reply by diana7127** [reply]
谢谢！问题已解决

**Comment by diana7127** [comment]


**Reply by hooke007** [reply]
没写错，建议手册看完整。选项很多，全列出来不现实（性能问题），你可以用自行编辑mpv.conf

**Comment by 19noozda** [comment]
Hello, why would you stop this magnificent project? :( it's hard to use those scripts without Ui.
Can you pls at least release new version Half- Yearly? MPV has no future unless someone Put UX +UI into it. mpv.net cm was that brilliant idea. 

WASTED

**Reply by hooke007** [reply]
https://github.com/hooke007/MPV_lazy/discussions/258#discussioncomment-4755032

**Comment by jolyonaha** [comment]
上游更新了

**Comment by MC-dusk** [comment]
绷，[上游诈尸了](https://github.com/mpvnet-player/mpv.net/issues/527#issuecomment-1681478558)
另外还有个[下游](https://github.com/diana7127/mpv.net-DW)（？）也在更新
没想到还有内销转出口的EN需求，看来除了语言，其他修改也很香啊

**Comment by hooke007** [comment]
这个项目不出意外我是不会更新了，上游作者仍在重点开发他的新项目。https://github.com/mpvnet-player/mpv.net/issues/527

mpv-lazy在转向uosc后我也没精力维护两个高度重合的项目

**Comment by sqq135** [comment]
我想反应一个原CM一个分支的字幕问题，但是好像只有合作者可以发言，正好看你也是，想知道是否可以转达。
https://github.com/diana7127/mpv.net-DW

转到lazy时，正好是向usco的开发者问https://github.com/hooke007/MPV_lazy/discussions/409 这个问题（开发者也给出了生效的回答）时，提到了字幕问题，复查出来是DW1.5改版后的问题。虽然已经准备换成lazy了，但毕竟用了DW一年多，所以想通过你反应这个bug。

https://github.com/tomasklaen/uosc/discussions/840

字幕问题
![image](https://github.com/hooke007/MPV_lazy/assets/78015121/3828a534-3bc5-4786-90c8-3a5ad8c3f277)

正常应该是这样（lazy、mpc、pot都是这样）
![image](https://github.com/hooke007/MPV_lazy/assets/78015121/d5f40e1e-3669-4f37-9b7f-f3fde91ab7a7)

问题版本：DW1.5原版重新安装
操作系统 win 11

[字幕文件，]([3-gatsu no Lion - 22.zip](https://github.com/hooke007/MPV_lazy/files/14045315/3-gatsu.no.Lion.-.22.zip))

**Reply by hooke007** [reply]
他这个项目不是有反馈区吗 https://github.com/diana7127/mpv.net-DW/discussions

**Reply by sqq135** [reply]
好像只对合作者开放，我也不太明白，找了半天按钮

![image](https://github.com/hooke007/MPV_lazy/assets/78015121/ba274a55-8522-48d2-b3d8-dec6af3e21b0)

**Reply by hooke007** [reply]
那直接@本人就完事了

**Reply by sqq135** [reply]
哈哈，我还不知道来着，那就@diana7127。

上面那两个不明所以的链接，是之前在你的评论区提的问题https://github.com/hooke007/MPV_lazy/discussions/409 ，然后https://github.com/tomasklaen/uosc/discussions/840 给解决了。

因为一个字幕问题，属实有点曲折了

**Reply by MC-dusk** [reply]
> 好像只对合作者开放，我也不太明白，找了半天按钮
> 
> ![image](https://private-user-images.githubusercontent.com/78015121/299505493-ba274a55-8522-48d2-b3d8-dec6af3e21b0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYxNTU3NjMsIm5iZiI6MTcwNjE1NTQ2MywicGF0aCI6Ii83ODAxNTEyMS8yOTk1MDU0OTMtYmEyNzRhNTUtODUyMi00OGQyLWIzZDgtZGVjNmFmM2UyMWIwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI1VDA0MDQyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEyNmQ3NTgwMjNlMDZhMjQwN2JlN2FhMzQ4ZWQzYjNmZDI0MTA1MjlhYWE1YzhjYTI4YTkxMTU2NzMwNzFlM2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Ll_AFApFMhILzWWjNCK11SFWbNcEvjzcBQ4XR6Ou96w)

我这里看是正常普通的：

![image](https://github.com/hooke007/MPV_lazy/assets/52311860/3fda8fcd-a339-4e5f-a264-3814aa8ce0bb)

你要不再试试，或者我帮你发，我1.8.0版本也有一样的问题

另外这个问题我测试了一下，估计是由于字幕在旋转前超出屏幕边界（这是本来是正常的，因为之后还会旋转），但程序自动换行防止其越界导致反复换行，纵向字幕本来离侧边界就近，你的例子里两个字就要换一次行，之后再旋转就变成一列两个字了。

**Reply by sqq135** [reply]
你要是能帮我发就更好了。经过测试，1.4是正常的，1.5、1.7、1.8都有问题，具体什么问题就不清楚了。同时net_CM和原版net（更新前和更新后）都没有这个问题

**Reply by diana7127** [reply]
v1.5-1.8版本为了实现次字幕使用了第三方内核，1.8之后的版本会换回官方内核

**Comment by vvincee7** [comment]
请问mpv net怎么反注册..在删除整个文件夹后，文件的打开方式里仍然存在无图标的mpv net，谢谢大佬

**Reply by MC-dusk** [reply]
嗯，虽然有点晚了，还是写一下。去注册表里面，直接搜索mpv，找到无效的mpvnet项删掉就行了，一般有几处，包括程序本身，几种特定格式的打开方式。或者使用[注册表清理器](https://apps.microsoft.com/detail/xpdls1xbtxvpp4)。因为我最近换版本也遇到了同样的问题。

**Reply by vvincee7** [reply]
感谢哈哈

**Comment by MC-dusk** [comment]
我想了半天总算搞明白为什么图都挂了，这个库原名`MPV_lazy`，改名`mpv_PlayKit`，至此都没有问题，因为github会自动重定向。
但是作者又建了一个新库起名`MPV_lazy`，本意是想手动重定向留个说明，但导致gjthub的自动重定向因为重名而失效，资源链接指向了新建的`MPV_lazy`库，这个库原本的图片、讨论在新库不存在就404了。

**Reply by hooke007** [reply]
gh 功能没你想的这么好😂，很多图在重建前就失效了，我是发现这点后才补的重名仓库
